docs/notebooks/constraints.txt

```diff
@@ -1,56 +1,63 @@
 absl-py==0.15.0
-aiohttp==3.7.4.post0
+aiohttp==3.8.1
```

Comment:
why so many changes? (out of curiosity)
is this generally OK? maybe a question for @uri-granta ?

Reply:
In our "requirements.txt" we do't fix most of the versions. So if between two runs of "generate_constraints" script our dependecies had minor releases (and many usually do), the script picks them up. That's as to "why".

It's not super convenient, I actually had a few errors that I fixed by rolling back some of the changes in this file. Perhaps fixing versions would be nicer. That's as to "is this OK".